packages/metals-vscode/package.json

```diff
@@ -343,6 +343,11 @@
           "default": false,
           "markdownDescription": "When this option is enabled, when user pastes any snippet into a Scala file, Metals will try to adjust the indentation to that of the current cursor."
         },
+        "metals.debug.hotCodeReplace": {
```

Comment:
MAybe we don't need a setting actually? If it doesn't work then the button would not work and that's fine as long it doesn't break anything else

Reply:
Okay, I'll remove the setting

Reply:
I don't think we need